### DISCUSSION TITLE
Testing Server

### DISCUSSION THREAD

**Discussion by Volte6** [GoMudEngine/GoMud#89] [Announcements]
If you want to test the mud without having to start up your own instance, you can do so at:

**TELNET**
HOST:`gomud.net` 
PORT: `33333`

**WEBCLIENT**
HOST `http://gomud.net/client`

I am particular interested in:

- If the server crashes/becomes unavailable, what did you do last?
- If something doesn't work the way you expected, or is confusing.
- If you have an idea for a feature that would be helpful/useful.

Also there is a super basic tutorial, i'm interested in how that can be improved.

**Comment by windu-ant** [comment]
I noticed a few things on the web client. The biggest of which is it appears to cause the tab to run out of memory and crash. I was able to have this occur twice. I killed a few rats, looked at the map a couple times, and wandered around the town. It slowly turned into a slideshow and then crashed.
![image_2024-09-24_232329242](https://github.com/user-attachments/assets/4d4e1fbd-53ec-4a94-ba4e-6c73481aec2a)
I have not tried using another browser yet.

![Screenshot 2024-09-24 231545](https://github.com/user-attachments/assets/e3d5ef66-0c55-40a4-a3ba-4ee798217874)
Not sure if this is a font issue on my end, but the map was not aligned.

![Screenshot 2024-09-24 231731](https://github.com/user-attachments/assets/2d2b24ba-546f-47df-a66e-ab3b233bb327)
And also this appears to have a display error if you have no skills.

**Reply by Volte6** [reply]
This should be fixed

**Comment by Volte6** [comment]
I've put in a small change that may fix the memory issue... I know what causes it.

You can also type "clear" tobclear the windows which should fix it in a pinch.

That rendering thing with the map is a font issue... your default monospace font isn't playing nice...I'll figure out something eventually, maybe I'll pick a font and embed it.

**Comment by windu-ant** [comment]
There is something that causes: "You're too encumbered to move!" however I do not see anything related to movement points or what exactly may be causing this. Seems to occur after moving quickly a bunch of rooms.

![image](https://github.com/user-attachments/assets/973d23cb-b02b-4b80-b4ec-dc43957004bf)

**Reply by Volte6** [reply]
It says you're carrying 5/5  in your inventory - but it shouldn't kick in until you hit 6/5. 

Also the messaging can be improved.

**Reply by Volte6** [reply]
I checked and that shouldn't be happening even at 5/5... and the messaging is reversed, it should be saying tired in your case. 

If it's limiting your movement too much It needs to be tuned better.

**Reply by windu-ant** [reply]
There is also a too tired message you get:
![image](https://github.com/user-attachments/assets/b6903b1c-1a62-42ca-bf16-b51ac13dd043)

**Reply by Volte6** [reply]
Yeah they were reversed.

But the question is how quickly do they deplete (too much?) You can add your movement points to your status bar to get a feel for how quickly it fills/depletes: `help prompt`:

<img width="943" alt="image" src="https://github.com/user-attachments/assets/4d5ccb64-3e34-436f-8098-6881f58881e9">

**Reply by Volte6** [reply]
encumbrance messaging should be improved. Still needs to be tweaked if it's too aggressively limiting your movement.

**Comment by windu-ant** [comment]
After buying a point of vitality my HP reset after some time:
![image](https://github.com/user-attachments/assets/872ae942-e2bd-4841-90ec-e93e4be5d0ee)

**Reply by Volte6** [reply]
Hmmm that's an interesting one... my guess is you had a buff applied, maybe you rested at the inn or ate something?

**Reply by windu-ant** [reply]
You are correct. I did get a buff from a mushroom. So it seems like its okay?

![image](https://github.com/user-attachments/assets/c5b530b9-6414-4712-a6db-78dfa2c16407)
![image](https://github.com/user-attachments/assets/aea11e4e-8ead-4fad-a399-625488e09283)
![image](https://github.com/user-attachments/assets/0313e8d8-a465-4553-a4e3-cfee61b075af)

**Reply by Volte6** [reply]
Messaging could be added for when buffs go away that affect your stats... the implications may be a lot more than just stuff you eat though, such as wearing/removing gear.

**Comment by windu-ant** [comment]
Your own stashes don't seem to be visible? I can recover an item from a stash but unless I remember the room myself I don't know where I put stuff, or I cannot find a way to look into the stash. Not sure if this is the intended behavior.

**Reply by Volte6** [reply]
"search" skill finds stashes.... I will add a feature that shows it as part of the items on the ground (stashed) if you own it.

**Reply by Volte6** [reply]
Stashed objects (by you) should appear in the item list now.

**Reply by Volte6** [reply]
<img width="1500" alt="image" src="https://github.com/user-attachments/assets/854ab8ca-f4f8-4b43-9edf-54f14335591e">
